assets/js/components/ChecksSelection/ChecksSelection.jsx

```diff
@@ -108,45 +92,13 @@ function ChecksSelection({
                     premium={check.premium}
                     selected={check.selected}
                     onChange={() => {
-                      setSelectedChecks(toggle(check.id, selectedChecks));
-                      onClear();
+                      onChange(toggle(check.id, selectedChecks));
                     }}
                   />
                 ))}
               </ChecksSelectionGroup>
             ))}
           </div>
-          <div className="place-items-end flex">
```

Comment:
What was the reason to move the buttons outside?

Reply:
Just positioning as per mockups, of course moving those to a completely different place I had to move the state declarations as well